radiotools/atmosphere/cherenkov_radius.py

```diff
@@ -0,0 +1,182 @@
+
+
+def get_cherenkov_radius_model_from_distance(zenith, d, obs_level, n0, model):
+    """ Calculates the radius of the (Cherenkov) cone with an apex at a given distance from ground 
+        along the shower axis with a given zenith angle. The open angle of the cone equals the 
+        Cherenkov angle for a value of the refractive index at this position.
+
+    Paramter:
+
+    zenith : double
+        Zenith angle (in radian) under which a shower is observed 
+    
+    d : double
+        Distance from ground to the apex, i.e., shower maximum of the observed shower along the shower axis (in m)
+    
+    obs_level : double
+        Altitude (in meter) of the plane at which the shower is observed 
+
+    n0 : double
+        Refractive index at sea level (!= obs_level)
+
+    model : int
+        Model index for the atmospheric (density) profile model.
+
+    Return : cherenkov Radius
+
+    """
+    height = atm.get_height_above_ground(
+        d, zenith, observation_level=obs_level) + obs_level
+    angle = get_cherenkov_angle_model(height, n0, model)
+    return cherenkov_radius(angle, d)
+
+
+def get_cherenkov_angle_model(height, n0, model):
+    """ Return cherenkov angle for given height above sea level, 
+        refractive index at sea level and atmospheric model. 
+
+    Paramter:
+
+    height : double
+        Height above sea level (in m)
+    
+    n0 : double
+        Refractive index at sea level (!= obs_level)
+
+    model : int
+        Model index for the atmospheric (density) profile model.
+
+    Return : cherenkov angle
+
+    """
+    n = atm.get_n(height, n0=n0, model=model)
+    return cherenkov_angle_model(n)
+
+
+def cherenkov_angle_model(n):
+    """ Return cherenkov angle for given refractive index.
+
+    Paramter:
+
+    n : double
+        Refractive index
+
+    Return : cherenkov angle
+
+    """
+    return np.arccos(1 / n)
+
+
+def cherenkov_radius(angle, d):
+    """ Return (cherenkov) radius
+
+    Paramters
+    ---------
+
+    angle : double
+        (Opening) angle of the cone (in rad)
+
+    d : double
+        Heigth of the cone (typically called distance, in meter)
+
+    Returns
+    -------
+    
+    radius : double
+        (Cherenkov) radius
+
+    """
+    return np.tan(angle) * d
+
+
+# old: cherenkov_angle_from_density_refractivity(rho, dxmax, n_asl, rho_0, ...)
+# param of the cherenkov angle from star-shape simulations
+# here cherenkov radius refers to radius of strongest emission
+# is used to determine rmax in the RdIdealGrid simulations!
+def cherenkov_angle_param(
+        height, dist, n0, model,
+        a=9.48990456e-01, b=4.48698860e+03,
+        c=1.43097665e+00, d=2.46630811e+06):
+
+    n = atm.get_n(height, n0=n0, model=model)
+    A = a - (b / dist) ** (c) - dist / d
+
+    return A * cherenkov_angle_model(n)
+# # old param
+# def cherenkov_angle_from_density(x, A=0.24905864, B=0.92165234):
+#     return np.deg2rad(A * np.log(x) + B)
```

Comment:
In my code I added `model` to the function names because I implemented the `revent` functions with the same name, I believe.

Anyway, I think it is a good idea to add them here.